README.rst

```diff
@@ -176,3 +176,6 @@ the `Tsunami programme`_ created by `sociomantic labs GmbH`_.
 .. _Tsunami programme: https://github.com/sociomantic-tsunami
 .. _sociomantic labs GmbH: https://www.sociomantic.com
 .. _Neptune: https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-user.rst#contributing-to-a-neptune-versioned-library
+.. |coveralls| image:: https://coveralls.io/repos/github/bokchan/libdrizzle-redux/badge.svg?branch=v6.2.x
+   :alt: Coveralls Build Status v6.2.x
+   :target: https://coveralls.io/github/bokchan/libdrizzle-redux?branch=v6.2.x
```

Comment:
Why not codecov, which is what we use for other Tsunami/OSS projects? /cc @mihails-strasuns-sociomantic

Reply:
> Why not codecov, which is what we use for other Tsunami/OSS projects? /cc @mihails-strasuns-sociomantic

Wasn't actually aware that we build coverage for our tsunami projects. 
I have no prior experience with continuous delivery of code coverage, so If `codecov` is as easy to integrate with travis as was the case with coveralls, I am fine with going with the tsunami flow.

Reply:
It should be even easier because it is already integrated into beaver: https://github.com/sociomantic-tsunami/ocean/blob/v3.x.x/.travis.yml#L38

Reply:
> It should be even easier because it is already integrated into beaver: https://github.com/sociomantic-tsunami/ocean/blob/v3.x.x/.travis.yml#L38

So fancy. I will go with that then

Reply:
BTW what's the builds which should be targeted with code-coverage. 
...I target the minor branches as they are updated after each PR has been merged

Reply:
There is no easy answer because we don't have fixed default branch in neptune-compatible repos. In existing tsunami repos we simply don't use build status badges so don't have that question :)

Reply:
Is there a shared codecov API key for tsunami projects or should a new one be generated for `libdrizzle-redux`?

Reply:
This is a @leandro-lucarella-sociomantic question

Reply:
We never needed an API key for codecov. I think is only required for private repos.

Reply:
Ah true. Forgot that. So it would only require setting up an extra repo for `libdrizzle-redux` at `codecov.io` under the `sociomantic-tsunami` username?